PyDSS/dssInstance.py

```diff
@@ -300,7 +308,7 @@ def RunStep(self, step, updateObjects=None):
         self._Logger.info(f'PyDSS datetime - {self._dssSolver.GetDateTime()}')
         self._Logger.info(f'OpenDSS time [h] - {self._dssSolver.GetOpenDSSTime()}')
         if self._Options['Profiles']["Use profile manager"]:
-            self.profileStore.update()
+            updated_values = self.profileStore.update()
```

Comment:
I don't see this being used. Should it?

Reply:
It is used by folks that use dssInstance directly. for example the NAERM effort